Arcade/TNT Floor/map.xml

```diff
@@ -0,0 +1,46 @@
+<map proto="1.4.0">
+<name>TNT Floor</name>
+<version>1.0.0</version>
+<objective>Shoot out the TNT under the other team to win!</objective>
+<gamemode>arcade</gamemode>
+<authors>
+    <authors> <author uuid="d1873b18-8194-429b-b6d5-eb7eb897071d"/> <!-- AVVERSION --> </authors>
+</authors>
+<contributors>
+    <contributor uuid="a9f11454-beac-4cc3-a138-1780802c8794" contribution="XML"/> <!-- Crazy_ --> </contributors>
+</contributors>
+<teams>
+    <team id="red" color="dark red" max="3">Red</team>
+    <team id="blue" color="blue" max="3">Blue</team>
+</teams>
+<kits>
+    <kit id="play">
+        <item slot="0" enchantment="arrow infinite;flame">bow</item>

```

Comment:
Make sure to make your important items unbreakable since people will be using it a lot.
```suggestion
        <item slot="0" unbreakable="true" enchantment="arrow infinite;flame">bow</item>
```